src/SFA.DAS.Campaign.UnitTests/Web/ViewComponents/WhenRenderingTheHeroHeadingViewComponent.cs

```diff
@@ -62,14 +71,16 @@ public void When_Apprentice_Then_The_Type_Is_Apprentice_And_Caption_Is_Apprentic
             var headingViewModel = (HeroHeadingViewModel)result.ViewData.Model;
 
             Assert.AreEqual(_sutModel.Type, headingViewModel.Type);
-            Assert.AreEqual(_sutModel.Caption, headingViewModel.Caption);
+            Assert.AreEqual(_sutModel.SectionTitle, headingViewModel.SectionTitle);
         }
 
         [Test]
         public void When_Employer_Then_The_Type_Is_Employer_And_Caption_Is_Employer()
         {
             _sutModel.Type = HeroHeadingType.Apprentice;
-            _sutModel.Caption = "Employer";
+            _sutModel.SectionTitle = "Employer";
+            _mockUrlHelper.Setup(url => url.Action(It.IsAny<UrlActionContext>())).Returns("www.apprenticeships.gov.uk/real-stories/employer");
```

Comment:
Does this URL make the test brittle?

Reply:
Turns out no as it wasn't needed for the test at all, so removed. :)